docs/configuration.rst

```diff
@@ -61,4 +84,28 @@ For other use cases, you can customize these configuration options in your ``con
 
    Type: bool
 
-   Notes: If this option is set to ``True``, the extension omits any prefix values from the URLs, including explicit values for ``notfound_default_language`` and ``notfound_default_version``.
+   .. note::
+
+      If this option is set to ``True``, the extension omits any prefix values from the URLs,
+      including explicit values for ``notfound_default_language`` and ``notfound_default_version``.
+
+   .. deprecated:: 0.5
+
+      ``notfound_no_urls_prefix`` is deprecated.  Use :confval:`notfound_urls_prefix` instead
+
+.. confval:: notfound_urls_prefix
+
+   Prefix added to all the URLs generated in the 404 page.
+
+   Default: ``'/en/latest/'``
```

Comment:
We should probably default to no prefix, I think.

Reply:
I'm not sure about this yet. However, defaulting to something different now, will create a breaking change.

Reply:
Default to no prefix makes the onboarding easier for projects outside Read the Docs. On the other hand `/en/latest/` makes it easier for RTD. So, there is a tradeoff there.

However, I wouldn't make this change without making a major version release. We can make that in another PR if we consider this change is better.